cities/berlin.json

```diff
@@ -934,8 +934,8 @@
             "properties": {
                 "title": "Wochenmarkt Bölschestraße",
                 "location": "Bölschestraße, 12587",
-                "opening_hours": null,
-                "opening_hours_unclassified": "Mo, Mi, Fr Sa 10:00 - 16:00 09:00 - 13:00"
+                "opening_hours": "Mo,We,Fr,Sa 10:00-16:00,09:00-13:00",
```

Comment:
It looks like the times are merged:

# Before

![image](https://user-images.githubusercontent.com/144518/145903961-1641ef66-a18e-4bb1-8209-7f090d01f8fa.png)

# After

![image](https://user-images.githubusercontent.com/144518/145903989-2172ae15-3f60-480d-a64e-cb32b12c963b.png)

This very likely isn't what the times should mean.

Reply:
I removed this code change from the branch.